### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Therealadetunji**
Add the mobile-menu to the mobile version of my portfolio.
using DOM manipulation in javascript and adding a javascript file to this project.

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Therealadetunji, This is @sja-thedude your code reviewer.

While you made a great effort in this project, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both partners in your pair programming group. Kindly make sure all members contribute.

Screenshot 2022-06-22 at 1 51 35 AM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const Close = document.querySelector(".Close");
+
+hamburger.onclick = () => {
+  //call-back function
+  navMenu.classList.add("mike");
+};
+Close.onclick = () => {
+  //call-back function
+  navMenu.classList.remove("mike");
+};
```
Comment (omar-muhamad) [inline script.js:15]: - [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears that's not happening So please fix that.

chrome-capture-2022-5-22

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add your collaborator name and social links in the Authors section in your README file to make it more professional.

image

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team,


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**